.dockerignore

```diff
@@ -0,0 +1,8 @@
+# Remove Git history - it doesn't needed in container
```

Comment:
What about `dist` folder?

Reply:
I don't understand the question. Do you suggest adding the dist folder here?

Reply:
Thanks for the suggestion. I implemented it